### ISSUE TITLE
Unit tests failing in oracle database

### ISSUE THREAD

**Issue by dpakach** [owncloud/customgroups#175]
when running unit tests in oracle database in drone this error occurs
```
1) OCA\CustomGroups\Tests\unit\CustomGroupsDatabaseHandlerTest::testGetGroupsByDisplayName
Failed asserting that two strings are equal.
--- Expected
+++ Actual
@@ @@
-'my_group_1'
+'my_group_3'
```

The drone link for this error is [here](https://drone.owncloud.com/owncloud/customgroups/6/53)

**Comment by PVince81** [comment]
hmmm, this sometimes happen if the unit tests are relying too much on database result order.
I might have missed doing this when writing them and it is known from experience that sometimes Oracle returns results out of order...

**Comment by PVince81** [comment]
@dpakach I assume you discovered this as part of your work in https://github.com/owncloud/customgroups/pull/173 ?

and perhaps we did not enable Oracle before so the issue was not discovered ?

in the future please link relevant tickets :-)

**Comment by dpakach** [comment]
@PVince81 yes, the tests which are in travis never used oracle so it might have missed before

**Comment by PVince81** [comment]
Fix is here: https://github.com/owncloud/customgroups/pull/187
